tests/integration/db_setup_js.go

```diff
@@ -57,17 +57,27 @@ func setupNode(
 		// no-op, use the `node` package default
 	}
 
-	node, err := node.New(s.Ctx, opts...)
+	nodeObj, err := node.New(s.Ctx, opts...)
 	if err != nil {
 		return nil, err
 	}
 	s.Ctx = acpIdentity.WithContext(s.Ctx, identity)
-	err = node.Start(s.Ctx)
+	err = nodeObj.Start(s.Ctx)
 	resetStateContext(s)
 	if err != nil {
 		return nil, err
 	}
-	c, err := setupClient(s, node)
+
+	// Get enableNAC boolean from the opts
+	tmpOptions := node.DefaultNodeACPOptions()
+	for _, opt := range opts {
+		if opt, ok := opt.(node.NodeACPOpt); ok {
+			opt(tmpOptions)
+		}
+	}
+	enableNAC := tmpOptions.IsEnabled()
```

Comment:
question: Why do you need to do this here?

In `tests/integration/client_setup_js.go`
```go
func setupClient(_ *state.State, node *node.Node, _ bool) (impl clients.Client, err error) {
	return js.NewWrapper(node)
}
```

The value is ignored anyway?